docs/contributing/source/ways-can-contribute/vanshdeep-notes.md

```diff
@@ -0,0 +1,163 @@
+# Vanshdeep's Notes on Running a Local Dev Node
+
+The following doc describes how to run a local node for developing BigchainDB Tendermint version.
+
+There are two crucial dependencies required to start a local node:
+
+- MongoDB
+- Tendermint
+
+and of course you also need to install BigchainDB Sever from the local code you just developed.
+
+
+## Installing MongoDB
+
+MongoDB can be easily installed, just refer their [installation documentation](https://docs.mongodb.com/manual/installation/) for your distro. 
+We know MongoDB 3.4 works with BigchainDB.
+MongoDB 3.6 _might_ work, or it might not. You could try it.
+After the installation of MongoDB is complete, run MongoDB using `sudo mongod --replSet=bigchain-rs`
+
+
+## Installing Tendermint
+
+### Installing Tendermint Using Docker
+
+Tendermint can be run directly using the docker image. Refer [here](https://hub.docker.com/r/tendermint/tendermint/) for more details.
+
+
+### Installing Tendermint from Source
```

Comment:
This section is really detailed, and I wonder if it's actually useful. If someone wants to build it from source, it's their responsibility to figure out how to install `go`, for example.

Reply:
This is intended for contributers who might need to consult Tendermint's source code or source code of Tendermint's dependencies.